.jscsrc

```diff
@@ -26,19 +26,17 @@
 	"excludeFiles": [
 		"**/node_modules/**",
 		"**/bower_components/**",
-		"**/generated/**",
-		"examples/ampersand/todomvc.bundle.js",
-		"examples/angular2/**/*.js",
+		"examples/vanilladart/**/*.js",
 		"examples/duel/www/**",
 		"examples/duel/src/main/webapp/js/lib/**",
-		"examples/humble/js/**",
-		"examples/js_of_ocaml/js/*.js",
+		"examples/ampersand/todomvc.bundle.js",
```

Comment:
looks like you reordered these (i am guessing in a rebase)

you should copy and paste this, and then add yours in the correct place :+1: 
https://github.com/tastejs/todomvc/blob/master/.jscsrc#L26-L41